### ISSUE TITLE
User cannot use keyboard navigation (tab button) to sign-in (Safari/Firefox) 

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9182]
## Observed behavior
When a user tries to log in from Safari via keyboard navigation, they can only sign in using the `Enter` button and not through tabbing.

![2022-03-14 11 48 51](https://user-images.githubusercontent.com/13563002/158240896-056557a5-7276-45fb-8c4e-2a2c00325189.gif)

## Errors and logs
None

## Expected behavior
User should be able to use both `Enter` button as well as `Tab` to navigate through sign-in page.

## User-facing consequences
It isn't possible to click on the `Explore without account` link or `Create account` via keyboard navigation.

## Steps to reproduce
1. Open in Safari
2. Use the `Tab` button to navigate the sign-in page

## Context
Kolibri 0.15
MacOS
Safari and Firefox

…

**Comment by marcellamaki** [comment]
Like #9194, I think this is Macbook settings dependent. After updating settings, tab navigation in the login seems to work correctly in both safari and firefox

![firefox-tab-login.gif](https://images.zenhubusercontent.com/6047f6e691103a78d3ad0ace/9af24c1a-2253-4395-b18e-e6081b769a1a)

![safari-tab-login.gif](https://images.zenhubusercontent.com/6047f6e691103a78d3ad0ace/a2b6d051-163b-421b-be20-06159c248eb3)

**Comment by marcellamaki** [comment]
cc @radinamatic -- to double check before closing (firefox)

**Comment by pcenov** [comment]
Confirmed - the Sign in page tab navigation works correctly on Safari, Firefox and Chrome on Mac. Had to enable this setting specifically for the Safari browser:
<img width="882" alt="safari-tab-to-highlight-each-item" src="https://user-images.githubusercontent.com/79847249/166415650-00520a75-ddd8-4e73-9fdc-718596be822f.png">
